src/snowflake/snowpark/mock/_functions.py

```diff
@@ -816,7 +823,7 @@ def mock_timestamp_ntz(
     result = _to_timestamp(column, fmt, try_cast, enforce_ltz=True)
     # Cast to NTZ by removing tz data if present
     return ColumnEmulator(
-        data=[x.replace(tzinfo=None) for x in result],
+        data=[x.replace(tzinfo=None) if x is not None else x for x in result],
```

Comment:
Can this be rewritten using `try_convert`?

Reply:
yes, it's already updated in another PR